js/charts/bullet.js

```diff
@@ -747,74 +860,109 @@ function bullet() {
 
                 _filterDimension[_dimension]._meta = {
                     dataType: _dimensionType[0],
-                    valueType: 'castValueType'
+                    valueType: "castValueType",
                 };
-                UTIL.saveFilterParameters(broadcast, filterParameters, parentContainer, _filterDimension, _dimension);
+                UTIL.saveFilterParameters(
+                    broadcast,
+                    filterParameters,
+                    parentContainer,
+                    _filterDimension,
+                    _dimension
+                );
             })
             .call(bullet);
 
-        plot.selectAll('.title').remove()
-        var title = _bullet.append('g')
-            .style('text-anchor', function (d) {
-                if (orientation == 'Horizontal') {
-                    return 'end';
-                } else if (orientation == 'Vertical') {
-                    return 'middle';
+        plot.selectAll(".title").remove();
+        var title = _bullet
+            .append("g")
+            .style("text-anchor", function (d) {
+                if (orientation == "Horizontal") {
+                    return "end";
+                } else if (orientation == "Vertical") {
+                    return "middle";
```

Comment:
Horizontal and Vertical should be in constant file

Reply:
done